### DISCUSSION TITLE
How are SV gnomAD annotations computed?

### DISCUSSION THREAD

**Discussion by drtconway** [broadinstitute/seqr#4353] [Q&A]
Hi seqr-people,

I'm trying to understand how SVs are annotated with population frequency. I have been grovelling through the GATK-SV code, and I can't manage to reconcile what I could see there, with the tooltip in seqr.

Are you able to point me to where I should be looking to understand how the seqr annotation is computed?

Thanks,
Tom.

**Comment by hanars** [comment]
seqr is not involved in SV annotation. SVs are not available for local installations at this time in part _because_ we rely on external groups to generate annotated data for us, we do not annotate our own data at this time so there is no code to point you to. You can fee free to reach out directly to the GATK-SV team if you have questions about that tool

**Reply by drtconway** [reply]
Thanks Hannah.
